xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
 
+<a name="Introduction"/>
+## 1.0 Introduction
 
-<a name="roleofxapi"/>
-## 1.0 Summary of the Experience API
+The Experience API (xAPI) is a technical specification that aims to facilitate the documentation and communication of learning experiences.  It does so by setting out a protocol that specifies how learning experiences should be described and an Application Programming Interface that sets out how information about these experiences should be stored and exchanged electronically.  
```

Comment:
Suggest splitting this second sentence in two as 'setting out an API that sets out' is not great. 

The specification does not set up how anything should be stored, just how it can be sent and retrieved.